src/components/Table.js

```diff
@@ -172,38 +172,39 @@ export default class Table extends React.Component<TableDefaultProps, TableProps
     }
   }
 
-  rowSelect(rowData: any, isSelected: boolean) {
-    const rowId = rowData.id;
+  rowSelect(rowData: any): boolean {
```

Comment:
So, when a row is clicked and the table is a multi-select one, we're toggling the selection state of that row, unless its currently selected and the `noEmptySelection` flag is set, in which case we don't change anything...

In the case of a single-select table, however, if the `noEmptySelection` is not set, it seems like the user can not deselect the selected row. Is that the case? And is that intentional?

Reply:
Also, I realize it's not part of this merge _per se_, but the comment for the `noEmptySelection` property is written as if it were the `noEmptySelection` property of the MasterDetails component instead of the Table component...
(Also, if we're saying that you can't ever deselect a row in a single-select table once one has been selected, then the `noEmptySelection` flag really only applies to multi-select tables and, for all intents and purposes, it's always true for single-select ones, and we should say that in the comment if we decide that this is how we want it to work...

Reply:
That wasn't my intention.
I'll fix it so that `noEmptySelection` is necessary to have this behavior in single-select as well.